src/components/network-area-diagram-viewer/network-area-diagram-viewer.ts

```diff
@@ -152,6 +152,25 @@ export class NetworkAreaDiagramViewer {
         return this.dynamicCssRules;
     }
 
+    private getNodeIdFromEquipmentId(equipmentId: string) {
+        const node: SVGGraphicsElement | null = this.container.querySelector(
+            'nad\\:node[equipmentid="' + equipmentId + '"]'
+        );
+        return node?.getAttribute('svgid') ?? '';
```

Comment:
not sure but i think returning null or undefined is more semantically appropriate than returning an empty string. This way, the caller can distinguish between a valid empty value and the absence of a value.

Reply:
Yes ok